client/state/rewind/retention/reducer.ts

```diff
@@ -0,0 +1,28 @@
+import {
+	JETPACK_BACKUP_RETENTION_UPDATE,
+	JETPACK_BACKUP_RETENTION_UPDATE_ERROR,
+	JETPACK_BACKUP_RETENTION_UPDATE_SUCCESS,
+} from 'calypso/state/action-types';
+import { BACKUP_RETENTION_UPDATE_REQUEST } from './constants';
+import type { AppState } from 'calypso/types';
+import type { AnyAction } from 'redux';
+
+const initialState = {
+	updateRequestStatus: BACKUP_RETENTION_UPDATE_REQUEST.UNSUBMITTED,
```

Comment:
Perhaps I didn't cover this well enough the last time, but don't see a good reason to need the form of the state to be an object with a `updateRequestStatus` field. `initialState` could directly hold the string value. We should just set it to default to  `BACKUP_RETENTION_UPDATE_REQUEST.UNSUBMITTED` and then in the reducer, we could set the value directly, and not to a property of an object. This would also simplify the reducer tests, and the selector (and its tests) when retrieving from this state tree.

Reply:
I was doing the same when addressing the feedback, but then I saw other rewind reducers', and all were having the  `requestStatus` as a sub-field, so decided to keep the structure similar in fashion. Also, we might add an action to retrieve the retention days separately (which is currently part of the other endpoint's response and needs refactoring) so as to add related reducers here later on.

Reply:
Other reducers keep that state because it's needed somewhere (whether it's really needed is a different topic). If it's not needed here, then we shouldn't add it. 

With regards to adding a different subtree in the future - that is a good excuse to keep this structure, but I've historically seen often that this doesn't happen, and the extra level remains there. My usual suggestion is to keep things simple and follow the YAGNI principle, and if we end up needing it, that's when we can refactor it.

Reply:
> Other reducers keep that state because it's needed somewhere (whether it's really needed is a different topic). If it's not needed here, then we shouldn't add it.

It's needed and will be used by @Initsogar upcoming PR to confirm updation.

>With regards to adding a different subtree in the future - that is a good excuse to keep this structure, but I've historically seen often that this doesn't happen, and the extra level remains there. My usual suggestion is to keep things simple and follow the YAGNI principle, and if we end up needing it, that's when we can refactor it.

Ok, will revert it and will add it again when needed :)

Reply:
Well, feel free to keep it if it's really needed and going to be used soon.

Reply:
Done, refactored the reducer to hold the status directly.

Reply:
> Well, feel free to keep it if it's really needed and going to be used soon.

Will segregate the update request status into a property when we refactor the other endpoint, for now, we can use the status directly as you suggested.